### ISSUE TITLE
FilesService deleteMany deletes no more than 100 elements

### ISSUE THREAD

**Issue by Enhed** [directus/directus#6560]
### Describe the Bug

if try delete programmicaly more than 100 elements - will be deleted just 100

### To Reproduce

```js
const { FilesService } = require("directus");
const fileService = new FilesService({ schema })
await fileService.deleteMany(items) // items have more than 100 elements

```

will be deleted just 100 elements

i think its cos inside method not uses limit option
![image](https://user-images.githubusercontent.com/36981278/123802706-410e2d00-d8f4-11eb-807f-09ee7d4cd1d6.png)

### What version of Directus are you using?

9.0.0-rc.82

### What version of Node.js are you using?

16.4.0

### What database are you using?

Postgres 12

### What browser are you using?

Chrome

### What operating system are you using?

Windows 10 x64

### How are you deploying Directus?

npm run start

**Comment by rijkvanzanten** [comment]
> i think its cos inside method not uses limit option

You're right! Wanna add a PR to add `limit: -1` there?

**Comment by Enhed** [comment]
> > i think its cos inside method not uses limit option
> 
> You're right! Wanna add a PR to add `limit: -1` there?

ill try)

**Comment by rijkvanzanten** [comment]
Fixed in #6561
